tests/search/test-search-dev-tools.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Automattic\VIP\Search;
+
+require_once __DIR__ . '/../../search/search.php';
+require_once __DIR__ . '/../../search/includes/classes/class-versioning.php';
+require_once __DIR__ . '/../../search/elasticpress/elasticpress.php';
+class Search_Dev_Tools_Test extends \WP_UnitTestCase {
+	/**
+	 * Make tests run in separate processes since we're testing state
+	 * related to plugin init, including various constants.
+	 */
+
+	// phpcs:disable WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase
+	protected $preserveGlobalState      = false;
+	protected $runTestInSeparateProcess = true;
+	// phpcs:enable
+
+	public function setUp() {
+		define( 'VIP_SEARCH_DEV_TOOLS', true );
+		$this->search_instance = new \Automattic\VIP\Search\Search();
+
+		require_once __DIR__ . '/../../search/search-dev-tools/search-dev-tools.php'; 
+	}
+
+	public function data_provider_endpoint_urls() {
+		return [
+			[
+				'input'    => 'http://vip-search:9200/vip-123-v1/_search',
+				'expected' => true,
+			],
+			[
+				'input'    => 'http://vip-search:9200/vip-123-v1/_search',
```

Comment:
Looks like this input is duplicated - did you mean to use a different endpoint than `_search`?

Reply:
Oh that meant to catch the multiple indexes, another failed copypasta.

Thanks for the catch!